recipe/yum_requirements.txt

```diff
@@ -0,0 +1 @@
+perl
```

Comment:
What is `perl` being used for?

Reply:
Uggh, yeah, so the runfile runs a perl script whose shebang line is `/usr/bin/perl`, so we can't even use conda-forge's perl. Thankfully, this only seems to be the case on linux.

Reply:
Ah ok. Thanks for the context. Yeah am ok with leaving this here. Was just confused initially as to where it was used. This makes sense :)